com.jetbrains.IntelliJ-IDEA-Ultimate.yaml

```diff
@@ -35,16 +35,16 @@ modules:
   - shared-modules/libsecret/libsecret.json
 
   - name: gcr
-    buildsystem: meson
```

Comment:
For the record, the rollback changes for `gcr` as part of this upgrade is because there was a direct commit into `master` before, so it needs to be reverted.